app/views/view_repayment_account.scala.html

```diff
@@ -68,7 +68,7 @@ <h1 id="main-message" class="govuk-heading-xl">@Messages("view_repayment_account
      </dd>
     </div>
    </dl>
-   <a href='@routes.BankAccountCocController.startBankAccountCocJourney(returnPage,audit)' id="call-bac">@Messages("view_repayment_account.change_account_details")</a>
+   <a class="govuk-body" href='@routes.BankAccountCocController.startBankAccountCocJourney(returnPage,audit)' id="call-bac">@Messages("view_repayment_account.change_account_details")</a>
```

Comment:
More recently I've taken to wrapping the main content section of my templates with <section class="govuk-body"> so that any nested p's are already styled

Reply:
Good idea, I'll give it a try